cmd/kube-controller-manager/app/plugins_providers.go

```diff
@@ -26,7 +26,6 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/pkg/volume"
 	"k8s.io/kubernetes/pkg/volume/azure_file"
-	"k8s.io/kubernetes/pkg/volume/azuredd"
```

Comment:
Please keep Azure cloud provider around.

Reply:
"k8s.io/legacy-cloud-providers/aws" is already removed here, are we sure we want to keep azuredd here?

Reply:
in L66, we want to remove azuredd in `appendAttachableLegacyProviderVolumes`, so we also need to remove the import azuredd here, right?

Reply:
Oh, sorry, please remove `azuredd`